components/operator/helm/values.yaml

```diff
@@ -26,6 +26,11 @@ operator:
   probeAddr: ":8081"
   # Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.
   enableLeaderElection: true
+  # Licence related configuration
+  licence:
+    token: ""
+    issuer: ""
+    secret: ""
```

Comment:
I think you don't need to provide the secret here.
I think you should create this secret using `token` in the helm chart